setup.cfg

```diff
@@ -87,6 +87,7 @@ exclude =
 
 [options.entry_points]
 console_scripts =
+    ctapipe = ctapipe.__main__:main
```

Comment:
There is no `main()` right now, which is why this fails to launch

Reply:
Ah, right, I locally only tested `python -m ctapipe`